pkg/cloud/amazon/common.go

```diff
@@ -69,10 +69,15 @@ func ResolveRegionWithoutOptions() (string, error) {
 func ParseContext(context string) (string, string, error) {
 	reg := regexp.MustCompile(`([a-zA-Z][-a-zA-Z0-9]*)\.((us(-gov)?|ap|ca|cn|eu|sa)-(central|(north|south)?(east|west)?)-\d)\.*`)
 	result := reg.FindStringSubmatch(context)
-	if len(result) < 3 {
-		return "", "", errors.Errorf("unable to parse %s as <cluster_name>.<region>.*", context)
+	arnReg := regexp.MustCompile(`arn:aws:eks:((us(-gov)?|ap|ca|cn|eu|sa)-(central|(north|south)?(east|west)?)-\d):[0-9]*:cluster/([a-zA-Z][-a-zA-Z0-9]*)`)
```

Comment:
Do you have a reference to the definition of the cluster arn. A link as a comment would be great and seeing it's definition would help to judge this regexp

Reply:
I've added a link in a comment in the code, not sure if thats where you want it.

- https://docs.aws.amazon.com/IAM/latest/UserGuide/list_amazonelasticcontainerserviceforkubernetes.html#amazonelasticcontainerserviceforkubernetes-resources-for-iam-policies
- https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html